packages/react/src/components/Dropdown/Dropdown.js

```diff
@@ -50,6 +50,7 @@ function Dropdown({
     itemToString,
     initialSelectedItem,
     onSelectedItemChange,
+    id: { id },
```

Comment:
I think this should just be `id,`. I'm seeing `[object Object]-label` as is

Reply:
Huge catch! Thanks for the assist and thanks @joshblack for throwing the commit in